docs/CONFIG.md

```diff
@@ -308,6 +294,40 @@ UseBatchSend = true # Default
 ```
 UseBatchSend toggles sending telemetry to the ingress server using the batch client.
 
+## TelemetryIngress.Endpoints]] # Example
+```toml
+[[TelemetryIngress.Endpoints]] # Example
+Network = 'EVM' # Example
+ChainID = '111551111' # Example
+ServerPubKey = 'test-pub-key-111551111-evm' # Example
+URL = 'http://localhost-111551111-evm:9000' # Example
```

Comment:
```suggestion
URL = 'localhost-111551111-evm:9000' # Example
```

Reply:
Fixed in [`73fcb44` (#10623)](https://github.com/smartcontractkit/chainlink/pull/10623/commits/73fcb4493c7c6cefc17708a35aeb6ac38665cd67)